config.json

```diff
@@ -809,6 +809,18 @@
       "topics": [
         "lists"
       ]
+    },
+    {
+      "slug": "simple-linked-list",
+      "uuid": "56a794de-40ee-4ff2-a1dd-f9b83f62d4b0",
+      "core": false,
+      "unlocked_by": "null",
```

Comment:
This should be unlocked by a core exercise, I'm open for suggestions.

Reply:
strain or accumulate ?

Reply:
I'm not very keen on the current order anyway.

I think we can safely put this under `accumulate` for now.

Though current structure is due to a change… And I think it will undergo some heavy changes over the next 6 months during the [big reorganisation project](https://exercism.io/blog/redesigning-tracks-in-partnership-with-chicago-university-and-sloan-foundation).